configs/base/index.js

```diff
@@ -7,33 +7,6 @@ module.exports = {
   rules: {
     // Additions
     '@typescript-eslint/consistent-type-imports': ['error'],
-    '@typescript-eslint/naming-convention': [
-      'error',
-      {
-        selector: 'default',
-        format: ['strictCamelCase'],
-        leadingUnderscore: 'forbid',
-        trailingUnderscore: 'forbid',
-      },
-      {
-        selector: 'typeLike',
-        format: ['StrictPascalCase'],
-      },
-      {
-        selector: 'variable',
-        // Exception for FunctionComponents
-        format: ['strictCamelCase', 'StrictPascalCase', 'UPPER_CASE'],
-      },
-      {
-        selector: 'function',
-        // Exception for FunctionComponents
-        format: ['strictCamelCase', 'StrictPascalCase'],
```

Comment:
This one is not in the new config, but I can't remember when this would be needed, so it's fine 👍